src/main/java/com/example/bookshop/dto/CreateBookRequestDto.java

```diff
@@ -1,14 +1,21 @@
 package com.example.bookshop.dto;
 
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
 import java.math.BigDecimal;
 import lombok.Data;
 
 @Data
 public class CreateBookRequestDto {
     private Long id;
+    @NotNull
     private String title;
+    @NotNull
     private String author;
+    @NotNull
     private String isbn;
+    @NotNull
+    @Min(0)
```

Comment:
Here also, we can use the `@PositiveOrZero` constraint since it is applicable for BigDecimals.